org.chrisoft.qmidiplayer.yml

```
@@ -0,0 +1,76 @@
+id: org.chrisoft.qmidiplayer
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
```

```suggestion
runtime-version: '5.15-23.08'
```

22.08 is getting EOL-ed this October